src/benchmark.cc

```diff
@@ -500,6 +504,11 @@ void BenchmarkImp::RangePair(int lo1, int hi1, int lo2, int hi2) {
   }
 }
 
+void BenchmarkImp::RangeMultiplier(int multiplier) {
+  CHECK_GE(multiplier, 2);
```

Comment:
it might make sense to add this check to `AddRange` instead where we already check the limits.